AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -45,6 +45,14 @@ @implementation MSAnalytics
 
 #pragma mark - Service initialization
 
++ (void)load {
```

Comment:
`load` called from first usage. It won't work correctly on start services one after another like in RN

Reply:
The load method of our classes are called only one time pre-main so it's really safe but that's because we are a static framework. I am not entirely sure about the behavior if one day we decide to ship a dynamic framework. Let's discuss options.

Reply:
The `load` method as we know in ObjC isn't supported in plain Swift: https://stackoverflow.com/questions/24898453/swift-objective-c-load-class-method. That doesn't make it future proof. Let's go back to migrating in the `init` method. Though we need to pay attention to the bug I mentioned about the need to  migrate before the `isEnabled` state is set in the abstraction.